virttest/utils_libvirt/libvirt_ceph_utils.py

```diff
@@ -0,0 +1,154 @@
+"""
+High-level libvirt ceph utility functions.
+
+This module is meant to reduce code size by performing common ceph setup procedures.
+:copyright: 2021 Red Hat Inc.
+"""
+
+import os
+import logging
+
+from avocado.utils import process
+from avocado.core import exceptions
+
+from virttest import data_dir
+from virttest import virsh
+from virttest import utils_package
+from virttest import ceph
+from virttest import utils_secret
```

Comment:
module names better be imported alphabetically.

Reply:
> module names better be imported alphabetically.

I observed there is guideline mentioning that import need to be categorized as system, thirty party ,and local .
I assume The import is more naturally arranged as where it is needed
Could you help share the guide or any reference that module names better be imported alphabetically ?

Reply:
Of course, I just learnt that knowledge a few weeks ago in 'Clean Code'. I'm sure you could find the info there. I won't paste the link here.

Reply:
updated